templates/flavors/rke2/rke2ControlPlane.yaml

```diff
@@ -33,11 +33,16 @@ spec:
   agentConfig:
     version: ${KUBERNETES_VERSION}
     nodeName: '{{ ds.meta_data.label }}'
+    cisProfile: ${CIS_PROFILE:-"cis-1.23"}
+    protectKernelDefaults: true
   preRKE2Commands:
     - |
       mkdir -p /etc/rancher/rke2/config.yaml.d/
       echo "node-ip: $(hostname -I | grep -oE 192\.168\.[0-9]+\.[0-9]+)" >> /etc/rancher/rke2/config.yaml.d/capi-config.yaml
     - sed -i '/swap/d' /etc/fstab
     - swapoff -a
     - hostnamectl set-hostname '{{ ds.meta_data.label }}' && hostname -F /etc/hostname
+    - cp -f /usr/local/share/rke2/rke2-cis-sysctl.conf /etc/sysctl.d/60-rke2-cis.conf
```

Comment:
should this be handled by the [RKEControlPlane](https://github.com/rancher-sandbox/cluster-api-provider-rke2/blob/69c69858ad6274fbd732bee5491f49dfddc81e3a/pkg/rke2/config.go#L67)?

Reply:
TIL, yeah lemme remove this

Reply:
Huh seems like it handles adding the etcd user too: https://github.com/rancher-sandbox/cluster-api-provider-rke2/blob/main/pkg/rke2/config.go#L53